src/Altinn.App.Api/Controllers/LookupOrganisationController.cs

```diff
@@ -16,19 +16,19 @@ namespace Altinn.App.Api.Controllers;
 [Consumes(MediaTypeNames.Application.Json)]
 [ResponseCache(Duration = 0, Location = ResponseCacheLocation.None, NoStore = true)]
 [Route("{org}/{app}/api/v1/organisations")]
```

Comment:
When I renamed the controller for person I also changed the route to be:
[Route("{org}/{app}/api/v1/lookup/person")]

That would mean this for organisation, if you agree on that URL. 
`[Route("{org}/{app}/api/v1/lookup/organisation")]`
We can also flip it around and have person/oranisation first, but I grouping them in the swagger doc under "Lookup" felt kind of nice.

Reply:
Hm, I'm not sure if I agree with this approach. According to REST a `{noun}/{id}` _is_ a lookup. Won't it be redundant then to specify `/lookup` in the URL?

Reply:
Yea, made more sense when it was in RPC form (aka a POST with a payload).
For person the URL isn't `{noun}/{id}`.

Reply:
The register API has the sensitive params as headers, which would allow us to not have a body on the person request, but figured that was more hassle for the frontend.

Reply:
Ah, that makes more sense. Although, I have believed it is more best practice to structure the urls like `{resource}/{action}`, like `person/lookup`, if we're following a REST paradigm (are we, or is it more RPC?). Although I can see that it's nice to group them in swagger 😄 But I might be wrong!

Reply:
Hahahah